GiniVision/Classes/CameraViewController.swift

```diff
@@ -95,7 +100,12 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
         guard let image = cameraOverlayImage, let cgImage = image.cgImage else {
             return nil
         }
-        return UIImage(cgImage: cgImage , scale: 1.0, orientation: UIDevice.current.orientation.isLandscape ? .right : UIImageOrientation.up)
+        if UIDevice.current.orientation.isFlat {
```

Comment:
On initialization there is not UIDeviceOrientation, so you need a reference to your current Orientation in order to set the correct orientation to cameraOverlay.